espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/exceptions/IncompatibleArgumentsException.java

```diff
@@ -0,0 +1,7 @@
+package io.appium.espressoserver.lib.handlers.exceptions;
+
+public class IncompatibleArgumentsException extends AppiumException {
```

Comment:
cannot we use the existing InvalidArgumentException instead?

Reply:
fixed